packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/vsphere-users.controller.js

```diff
@@ -21,10 +22,12 @@ export default class DedicatedCloudVsphereUsersCtrl {
         federationStatus: this.DedicatedCloud.getFederationStatus(
           this.productId,
         ),
+        iamStatus: this.DedicatedCloud.getIamStatus(this.productId),
         policy: this.DedicatedCloud.getPasswordPolicy(this.productId),
       })
       .then((response) => {
         this.federationEnabled = response.federationStatus?.state === 'enabled';
+        this.iamEnabled = response.iamStatus?.state === 'enabled';
```

Comment:
**nitpick** Can be maintained in the const file

Reply:
Same as https://github.com/ovh/manager/pull/9649#discussion_r1315581881

Reply:
Ok, done